### PR TITLE
Update Ibeji Dockerfiles

### PR DESCRIPTION
**Pull request by devkelley**
This PR does the following:

- Removes 'containerize' feature: commit 1 (e2a4d10)
- Add aarch64 target configuration: commit 1 (e2a4d10)
- Update/Add Dockerfiles for arm64 and amd64. Dockerfiles are now more generic: commit 2 (26e106b)
- Update container READMEs: commit 3 (2ab7bf4)

### CODE REVIEW

**Review by wilyle — DISMISSED** [review]

File: container/README.md [hunk 1]
```diff
@@ -1,119 +1,271 @@
...
 
-1. Run the following command in the project's root directory to build the docker container from the
+>Note: Before running any of the following commands, replace all placeholders (wrapped with `<>`).
+
+1. Run the following command in the project root directory to build the docker container from the
 Dockerfile:
 
     ```shell
-    docker build -t invehicle_digital_twin -f Dockerfile .
+    docker build -t <image_name> -f <Dockerfile> (optional: --build-arg=APP_NAME=<project name>) (optional: --build-arg=FEATURES="<module name(s)>") .
```
[the remark this answers is not in the document]
Comment (devkelley) [reply container/README.md:63]: Okay will do.

[the remark this answers is not in the document]
Comment (devkelley) [reply container/README.md]: updated to be more concise

**Review by wilyle — APPROVED** [review]

**Review by ashbeitz — DISMISSED** [review]
Approved

**Review by ashbeitz — APPROVED** [review]
Approved

**Pull request merged**